cool_seq_tool/mappers/mane_transcript.py

```diff
@@ -1005,3 +1022,93 @@ async def g_to_mane_c(
                 ensembl_c_ac=current_mane_data["Ensembl_nuc"],
                 alt_ac=grch38["ac"] if grch38 else None,
             )
+
+    async def grch38_to_mane_p(
+        self,
+        alt_ac: str,
+        start_pos: int,
+        end_pos: int,
+        gene: Optional[str] = None,
+        residue_mode: ResidueMode = ResidueMode.RESIDUE,
+        try_longest_compatible: bool = False,
+    ) -> Optional[Dict]:
+        """Given GRCh38 genomic representation, return protein representation.
+        Will try MANE Select and then MANE Plus Clinical. If neither is found and
+        `try_longest_compatible` is set to `true`, will also try to find the longest
+        compatible remaining representation.
```

Comment:
Fwiw, sometimes Sphinx API page generation bugs out if you don't include an extra blank line between the descriptions and the params, so I've been trying to include them when I remember (for public methods)

Reply:
Cool. I usually add a new line, but must've forgot for this one